ios/Podfile.lock

```diff
@@ -859,7 +859,7 @@ SPEC CHECKSUMS:
   CodePush: 9bb93f52a44ee80b72739d89dead9c7e86c3bd3a
   DoubleConversion: cf9b38bf0b2d048436d9a82ad2abe1404f11e7de
   FBLazyVector: e686045572151edef46010a6f819ade377dfeb4b
-  FBReactNativeSpec: 482cfbfb1c36db6e16e230b2df6af5bb132a8177
+  FBReactNativeSpec: 81ba71db3062ee7fae63b8c55a5dad186f8728cf
```

Comment:
dont like this change - last time I spend all night to fix builds after update this, not sure why this coming and fill is some internal problem with RN, we need test ios after merge this

Reply:
done, reverted